src/client/cypress/e2e/admin/userAdministration.cy.js

```diff
@@ -1,8 +1,13 @@
-import { interceptApiCalls, loginAsAdmin } from "../testHelpers";
+import {
+  interceptApiCalls,
+  loginAsAdmin,
+  loginAndResetBoreholes,
+} from "../testHelpers";
 
 describe("Admin settings test", () => {
   beforeEach(() => {
     interceptApiCalls();
+    loginAndResetBoreholes();
```

Comment:
REQ: `loginAndResetBoreholes()` und `interceptApiCalls()` sollen in einen [Root-Level-Hook](https://github.com/mochajs/old-mochajs-site/blob/master/index.md#root-level-hooks) bzw. [Support-File](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Support-file) ausgelagert werden, so sparen wir uns den Aufruf in jedem einzelnen Test.